src/adapter/debugAdapter.ts

```diff
@@ -106,6 +106,9 @@ export class DebugAdapter {
 
   async _onSetBreakpoints(params: Dap.SetBreakpointsParams): Promise<Dap.SetBreakpointsResult | Dap.Error> {
     const ids = generateBreakpointIds(params);
+    setTimeout(() => {
+       throw new Error('aweffeaw');
+    }, 10000);
```

Comment:
🔥

Reply:
Thanks for catching this... This is how I was testing the code xD

Reply:
Done